### PR TITLE
fix: update publish command and add notice to raise PR against next branch

### PR DESCRIPTION
**Pull request by rajeshtezu**
# COMPLETES #<[SPARK-449056](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-449056)>

## This pull request addresses

- Updates the publish command in each `package.json` file under packages
- Add notice to announce `master` branch will be frozen and to raise all the PRs against `next` branch

## by making the following changes

Screenshot 2023-08-23 at 8 32 06 PM

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [x] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [ ] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [ ] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Contributing
 
+<div style="display: inline-block; padding: 4px 8px; background-color: yellow; font-size: 1.5rem; margin-bottom: 1rem; border-radius: 8px">
+  <strong>NOTE</strong>
+  <ul>
+    <li style="margin: 0"><strong>master</strong> branch will be frozen and no further releases will go out on that branch.</li>
```
Comment (sreenara) [inline CONTRIBUTING.md:6]: Could we reword the text to the following:
The 'master' branch is being locked, and there won't be any more releases from that branch. Kindly initiate new pull requests against the 'next' branch instead.

**Review by sreenara — CHANGES_REQUESTED** [review]
Just a comment regarding the text

File: CONTRIBUTING.md (hunk 1 above)
Comment (rarajes2) [reply CONTRIBUTING.md]: Updated the text

**Review by mkesavan13 — APPROVED** [review]

**Review by InteractiveTimmy — CHANGES_REQUESTED** [review]
In order for this to publish within a workspace environment, the command on each package must used `yarn`:

```bash
yarn npm publish
```

**Comment by rarajes2** [comment]
> In order for this to publish within a workspace environment, the command on each package must used `yarn`:
> 
> ```shell
> yarn npm publish
> ```

Done

**Pull request merged**